internal/cmd/stack/stack_selector.go

```diff
@@ -7,18 +7,33 @@ import (
 
 	"github.com/manifoldco/promptui"
 	"github.com/pkg/errors"
+	"github.com/shurcooL/graphql"
+	"github.com/spacelift-io/spacectl/internal/cmd/authenticated"
```

Comment:
Sorted imports are nice imports: https://github.com/spacelift-io/spacectl/blob/main/internal/cmd/run_external_dependency/mark_completed.go#L3

Reply:
What do you mean by that? I've got autoformatting and linting enabled in my editor and these imports seem to be in lexicographic order.

Reply:
That's fine, but not quite right. The default accepted way of ordering imports is:
```
STD library

external dependencies

internal dependencies
```

I think [goimports](https://pkg.go.dev/golang.org/x/tools/cmd/goimports) should do that by default actually.

Reply:
ah, thx

for future reference, it was just a matter of toggling an option in code style settings:

![Screenshot from 2023-09-29 11-38-17](https://github.com/spacelift-io/spacectl/assets/12897831/68f2e2f2-1afb-4962-8ed4-b83f45304a65)